.github/workflows/tests.yml

```diff
@@ -108,15 +112,89 @@ jobs:
           tfsec_return="${{ steps.test.outputs.tfsec-return-code }}"
           reviewdog_return="${{ steps.test.outputs.reviewdog-return-code }}"
 
-          if [ "$tfsec_return" -eq 1 ]; then
+          if [[ "$tfsec_return" -eq 1 ]]; then
+            echo "tfsec correctly returned failure ${tfsec_return}"
+          else
+            echo "tfsec returned ${tfsec_return}, expected '1'. Failing..."
+            exit 1
+          fi
+
+          if [[ "$reviewdog_return" -eq 0 ]]; then
+            echo "reviewdog correctly returned success: ${reviewdog_return}"
+          else
+            echo "reviewdog returned ${reviewdog_return}, expected '0'. Failing..."
+            exit 1
+          fi
+
+  test-macos:
+    name: tfsec (macos)
```

Comment:
Can you use matrix instead?
example: https://github.com/reviewdog/action-golangci-lint/blob/db0dd9927fe4eebeab6eda0f066f939ee4d132c5/.github/workflows/reviewdog.yml#L22-L24

Reply:
Done in https://github.com/reviewdog/action-tfsec/pull/15/commits/fe2837ffee1759ab1e67f1ae2c8bf775ad113dd1!